definitions/GarrPlotUICategory.dbd

```diff
@@ -1,12 +1,11 @@
 COLUMNS
-uint field_329935761?
 locstring categoryName_lang
 uint field_2890979807?
 int ID
 uint plotType
 
 BUILD 6.0.1.18179
-field_329935761<32>
+$noninline,id$ID<32>
```

Comment:
This is the only noninline in this diff and version. Are you sure about that?

Reply:
For the record, this isn't noninline, previous build issue again.